modules/api/src/main/java/org/apache/ignite/configuration/schemas/runner/LocalConfigurationSchema.java

```diff
@@ -15,23 +15,32 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.configuration.extended;
+package org.apache.ignite.configuration.schemas.runner;
 
 import org.apache.ignite.configuration.annotation.ConfigValue;
 import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.Value;
 import org.apache.ignite.configuration.storage.ConfigurationType;
 
 /**
- *
+ * Local node configuration schema.
  */
 @SuppressWarnings("PMD.UnusedPrivateField")
 @ConfigurationRoot(rootName = "local", type = ConfigurationType.LOCAL)
 public class LocalConfigurationSchema {
-    /** */
+    /** It is a copy of appropriate property from the cluster configuration. */
+    @Value(hasDefault = true)
+    String[] metastorageMembers = new String[0];
+
+    /**
+     *
+     */
     @ConfigValue
     private BaselineConfigurationSchema baseline;
```

Comment:
Seems that we don't need given field here. BaselineConfigutation will have ConfiguarionType.DISTIRBUTED.

Reply:
Removed.